### ISSUE TITLE
Deleting media does not deletes subtitles

### ISSUE THREAD

**Issue by pacoccino** [meeb/tubesync#416]
When medias are deleted, such as when removing a source, the subtitles are not removed. 

We should check if the source has enabled subtitles and delete all `{filename}.vtt` files too

**Comment by meeb** [comment]
Thanks, should be relatively easy to handle in the delete signal. Is it just `*.vtt` files?

**Comment by pacoccino** [comment]
Yes, just vttLe 8 oct. 2023 à 06:08, meeb ***@***.***> a écrit :﻿
Thanks, should be relatively easy to handle in the delete signal. Is it just *.vtt files?
